.github/workflows/tag_action.yml

```diff
@@ -3,25 +3,68 @@ name: Publish release
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+**'
 
 jobs:
   build:
     permissions: write-all
-    name: Add documentaiton as asset
+    name: Create release page
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
-      - name: Build
+
+      - name: Check artifacts from commit
+        id: get_artifacts
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          commit: ${{github.sha}}
+          workflow: on_commit.yml
+          dry_run: true
+          if_no_artifact_found: ignore
+      
+      - name: Sleep 15 min to wait for builds
```

Comment:
Shouldn't we start that action in flow after the build instead of waiting fixed amount of time?

Reply:
This step is executed only if the build did not finish yet. If the artifacts were found in previous step, this sleep is omited.

It would be nice to trigger this action automatically is that the tag can be created at any time.
So we can not trigger this action after the commit build has finished (because we do not know if the tag has been created on this commit) 

So the current trigger is the most accurate - This action will be executed when the tag is pushed to repo, but if we do merge, and quickly push the tag, the on commit plan will not finish yet, and the artifacts will be missing. In that case waiting 15min, is no such long time, and it ensures that the build finishes ( it takes <10 min)

Reply:
sound reasonable
maybe there is a better solution, but lets give this one a try - if it fails we will investigate for the better one